common/resource/resourceTest.go

```diff
@@ -176,6 +177,7 @@ func NewTest(
 		// other common resources
 
 		DomainCache:       cache.NewMockDomainCache(controller),
+		MetricsScopeCache: cache.NewMetricsScopeCache(),
```

Comment:
I think it's probably better to use a real implementation rather than a mock here. Otherwise we will have to specify many mock calls for this cache in the test.